src/Symfony/Component/Security/Core/Test/Authentication/ConcreteToken.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Security\Core\Test\Authentication;
+
+use Symfony\Component\Security\Core\Authentication\Token\AbstractToken;
+use Symfony\Component\Security\Core\User\UserInterface;
+
+/**
+ * A very limited token that is used to login in tests using the KernelBrowser.
+ *
+ * @author Wouter de Jong <wouter@wouterj.nl>
+ */
+class ConcreteToken extends AbstractToken
```

Comment:
I'm not sure I like the token name ... its description is very clear to me so, what about naming it more closely to its description? E.g. `TestingToken`, `TestLoginToken`, etc.

Reply:
I've renamed it to `TestKernelToken` (that's the tl;dr version of the description I think). I've also moved this to the FrameworkBundle and reverted removing the `ConcreteToken` of the `AbstractTokenTest`, as it's name now very much relates to the `KernelBrowser` in the FrameworkBundle. This also allowed me to remove the weird `$credentials` things in this class.